bo/bo_loop.py

```diff
@@ -261,6 +261,18 @@ def run(self):
                 #print("kg_val ", kgvalue)
 
             new_x_ckg, acqf_value_ckg = self.get_best_coupled_kg_value(best_observed_location, best_observed_value, iteration, model)
+
+            posterior = model.posterior(new_x)
```

Comment:
I think we already have a method to compute this stuff. If you call ConstrainedPosteriorMean, one of the methods computes the feasibility of each individual constraint